app/src/main/java/info/nightscout/androidaps/receivers/KeepAliveWorker.kt

```diff
@@ -191,7 +191,7 @@ class KeepAliveWorker(
         }
         if (loop.isDisconnected) {
             // do nothing if pump is disconnected
-        } else if (runningProfile == null || ((!pump.isThisProfileSet(requestedProfile) || !requestedProfile.isEqual(runningProfile) || (runningProfile is ProfileSealed.EPS && runningProfile.value.originalEnd < dateUtil.now())) && !commandQueue.isRunning(Command.CommandType.BASAL_PROFILE))) {
```

Comment:
Simply removing te code will return the previous issue

Reply:
Thinking of it it's better to revert the workaround for this PR and address https://github.com/nightscout/AndroidAPS/issues/2567 separately